metricbeat/docs/configuring-howto.asciidoc

```diff
@@ -67,7 +71,9 @@ metricbeat:
       selectors: []
```

Comment:
We need to remove selectors here and above. selectors does not exist anymore :-(